src/client/index.ts

```diff
@@ -19,26 +103,82 @@ export function removeEventListener<E extends NotificationEvent>(eventType: E['t
     eventEmitter.removeListener(eventType, listener);
 }
 
+/**
+ * Creates a new Notification.
+ *
+ * The Notification will appear in the Notification Center or as a toast if the Center is not visible.
+ *
+ * ```ts
+ * import {create} from 'openfin-notifications';
+ *
+ * create({
+ *      id: "uniqueNotificationId",
+ *      body: "I'm the Notification body text",
+ *      icon: "https://openfin.co/favicon.ico"
+ * });
+ * ```
+ *
+ * @param options Notification configuration options.
+ */
 export async function create(options: NotificationOptions): Promise<Notification> {
     // Should have some sort of input validation here...
     return tryServiceDispatch(APITopic.CREATE_NOTIFICATION, options);
 }
 
+/**
+ * Clears a specific Notification from the Notification Center.
+ *
+ * ```ts
+ * import {clear} from 'openfin-notifications';
+ *
+ * clear("uniqueNotificationId");
+ * ```
+ *
+ * @param id ID of the Notification to clear.
+ */
 export async function clear(id: string): Promise<boolean> {
     // Should have some sort of input validation here...
     return tryServiceDispatch(APITopic.CLEAR_NOTIFICATION, {id});
 }
 
+/**
+ * Retrieves all Notifications which were generated by your applications uuid.
+ *
+ * ```ts
+ * import {getAll} from 'openfin-notifications'
+ *
+ * getAll()
+ *  .then(console.log);
+ * ```
+ */
 export async function getAll(): Promise<Notification[]>{
     // Should have some sort of input validation here...
     return tryServiceDispatch(APITopic.GET_APP_NOTIFICATIONS, undefined);
 }
 
+/**
+ * Clears all Notifications which were generated by your applications uuid.
```

Comment:
"application's" (or do something similar to Tom's comment above on a similar comment)